src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Raw.cs

```diff
@@ -173,12 +249,10 @@ private void WriteRawValueCore(ReadOnlySpan<byte> utf8Json, bool skipInputValida
             {
                 // Utilize reader validation.
                 Utf8JsonReader reader = new(utf8Json);
-                while (reader.Read());
+                while (reader.Read()) ;
                 _tokenType = reader.TokenType;
             }
 
-            // TODO (https://github.com/dotnet/runtime/issues/29293):
-            // investigate writing this in chunks, rather than requesting one potentially long, contiguous buffer.
```

Comment:
Curious why this got removed? Was this addressed here?

Reply:
Added back per [comment](https://github.com/dotnet/runtime/pull/76444#discussion_r987857215)